tests/test-cases/Group1-Docker-Commands/1-13-Docker-Version.robot

```diff
@@ -18,10 +18,15 @@ Docker Version Format Client Version
     Should Be Equal As Integers  ${rc}  0
     Should Not Be Empty  ${output}
 
-Docker Version Format Client API Version
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} version --format '{{.Client.APIVersion}}'
+Docker1.11 Version Format Client API Version
+    ${rc}  ${output}=  Run And Return Rc And Output  docker1.11 %{VCH-PARAMS} version --format '{{.Client.APIVersion}}'
     Should Be Equal As Integers  ${rc}  0
-    Should Not Be Empty  ${output}
+    Should Be Equal As Integers  ${output} 1.23
```

Comment:
${output} 1.23--> should it be ${output}   1.23? I mean double white-space between output and 1.23.